tests/testutils.py

```diff
@@ -30,9 +30,10 @@ def tearDown(self):
         super().tearDown()
         registry.clear()
 
-        from django_components.template import _create_template
+        from django_components.template import template_cache
 
-        _create_template.cache_remove()  # type: ignore[attr-defined]
+        if template_cache:
```

Comment:
Is this if needed here?

Reply:
Yeah, I just checked, and there are two tests in `tests/test_component.py` that break otherwise. I've added a comment.

Reply:
No, I meant just the if. We’re importing a module, and then saying ”if this module is truthy”? What does that add?

Reply:
Ah, right 😄 The cache is initiated lazily, because it needs info from the settings. [At the beginning the variable may be None](https://github.com/JuroOravec/django-components/blob/a1bd29f7a8fc63fb8c183d1e3f52477b670ff485/src/django_components/template.py#L13)